src/XIVLauncher.Common.Unix/Compatibility/DxvkSettings.cs

```diff
@@ -1,121 +1,132 @@
 
-    public DxvkSettings(Dxvk.DxvkHudType hud = Dxvk.DxvkHudType.None, string dxvkHudCustom="",
-        string mangoHudPath="", bool? async = true, int? frameRate = 0,
-        Dxvk.DxvkVersion version = Dxvk.DxvkVersion.v1_10_1, DirectoryInfo? corePath = null)
+    public DxvkSettings(Dxvk.DxvkHudType hud, DirectoryInfo corePath, Dxvk.DxvkVersion version, bool enabled = true, string? dxvkHudCustom = null, FileInfo? mangoHudConfig = null, bool async = true,
+                        int maxFrameRate = 0)
     {
-        this.DxvkHud = hud;
-        string home = Environment.GetEnvironmentVariable("HOME");
-        corePath ??= new DirectoryInfo(Path.Combine(home,".xlcore"));
-        var dxvkConfigPath = new DirectoryInfo(Path.Combine(corePath.FullName,".dxvk"));
+        Enabled = enabled;
+        DxvkHud = hud;
+        var dxvkConfigPath = new DirectoryInfo(Path.Combine(corePath.FullName, "compatibilitytool", "dxvk"));
         if (!dxvkConfigPath.Exists)
             dxvkConfigPath.Create();
-        this.DxvkVars = new Dictionary<string, string> ();
-        this.DxvkVars.Add("DXVK_LOG_PATH",Path.Combine(corePath.FullName,"logs"));
-        this.DxvkVars.Add("DXVK_CONFIG_FILE",Path.Combine(dxvkConfigPath.FullName,"dxvk.conf"));
-        this.DxvkVars.Add("DXVK_ASYNC", ((async ?? false) ? "1" : "0"));
-        frameRate ??= 0;
-        if (frameRate > 0) this.DxvkVars.Add("DXVK_FRAME_RATE", (frameRate).ToString());
-        this.DxvkVersion = version;
-        string release = this.DxvkVersion switch
+        DxvkVars = new Dictionary<string, string>
+        {
+            { "DXVK_LOG_PATH", Path.Combine(corePath.FullName, "logs") },
+            { "DXVK_CONFIG_FILE", Path.Combine(dxvkConfigPath.FullName, "dxvk.conf") },
+            { "DXVK_ASYNC", async ? "1" : "0" },
+            { "DXVK_FRAME_RATE", (maxFrameRate).ToString() }
+        };
+        DxvkVersion = version;
+        var release = DxvkVersion switch
         {
             Dxvk.DxvkVersion.v1_10_1 => "1.10.1",
             Dxvk.DxvkVersion.v1_10_2 => "1.10.2",
             Dxvk.DxvkVersion.v1_10_3 => "1.10.3",
             Dxvk.DxvkVersion.v2_0 => "2.0",
             _ => throw new ArgumentOutOfRangeException(),
         };
-        this.DownloadURL = $"https://github.com/Sporif/dxvk-async/releases/download/{release}/dxvk-async-{release}.tar.gz";
-        this.FolderName = $"dxvk-async-{release}";
-        string dxvkCachePath = Path.Combine(dxvkConfigPath.FullName, release + ((async ?? false) ? "-async" : "")); 
+        DownloadURL = $"https://github.com/Sporif/dxvk-async/releases/download/{release}/dxvk-async-{release}.tar.gz";
+        FolderName = $"dxvk-async-{release}";
+        string dxvkCachePath = Path.Combine(dxvkConfigPath.FullName, release + (async ? "-async" : ""));
         this.DxvkVars.Add("DXVK_STATE_CACHE_PATH", dxvkCachePath);
```

Comment:
Create a cache folder, and then put the various state caches in those.
```suggestion
        DirectoryInfo dxvkCachePath = new DirectoryInfo(Path.Combine(dxvkConfigPath.FullName, "cache"));
        if (!dxvkCachePath.Exists) dxvkCachePath.Create();
        this.DxvkVars.Add("DXVK_STATE_CACHE_PATH", Path.Combine(dxvkCachePath.FullName, release + (async ? "-async" : "")));
```

The folder tree will look like this:
```
compatibilitytool
├── beta
│   └── wine-xiv-staging-fsync-git-7.10.r3.g560db77d
└── dxvk
    ├── cache
    │   └── 1.10.3-async
    │   └── 2.0
    ├── dxvk-async-1.10.1
    ├── dxvk-async-1.10.3
    └── dxvk-async-2.0

```